Cargo.toml

```diff
@@ -3,8 +3,9 @@ name = "zt-tcp-relay"
 version = "0.1.0"
 authors = ["Alexander Akhmetov <me@aleks.sh>"]
 edition = "2018"
+license = "MIT"
```

Comment:
I hope that's ok. It's a popular license choice in the Rust ecosystem.
Without a license one cannot package this for Linux distributions.